akka-http-tests/src/test/scala/akka/http/scaladsl/server/EntityStreamingSpec.scala

```diff
@@ -278,6 +279,13 @@ class EntityStreamingSpec extends RoutingSpec with ScalaFutures {
               .runFold(0) { (cnt, _) => cnt + 1 }
 
           complete {
+            implicit val marshaller = // Scala 3 workaround for missing implicit conversion
+              Marshaller.futureMarshaller(
+                Marshaller.liftMarshaller(
+                  sprayJsonMarshaller(mapFormat(StringJsonFormat, StringJsonFormat))
+                )
+              )
+
```

Comment:
I (fruitlessly) experimented for a bit, but ended up just writing the marshaller explicitly since nothing else seemed to work for both Scala versions (I got this via scala.reflection reify from scala 2.13 compilation).